wc-i18n.html

```diff
@@ -77,6 +77,13 @@
   Polymer({
     is: 'wc-i18n',
 
+    /**
+     * Fired when the requested locale/key combo does not exist.
+     * Exposes an object with locale and key properties.
+     *
+     * @event wc-i18n:locale-not-found
```

Comment:
For `v2` of this component this event name has been updated. Please refactor to that and this is good to go.